iterators_task_1/student/include/impl.h

```diff
@@ -10,4 +10,14 @@
  *
  * @warning YOU MUST USE STRING STREAMS AND STREAM ITERATORS!
  */
-std::string reverseWords(const std::string& input);
+std::string reverseWords(const std::string& input) {
+    std::stringstream myStream(input);
+    std::string token, reversedSentence;
+    while(myStream >> token) {
```

Comment:
1. Use std::copy
2. Or initialize container with pair of stream iterators (begin is pointing to the stream, second is default-initialized)
3. Use reverse iterators/prepend to the container